lib/src/screens/supa_email_auth.dart

```diff
@@ -53,8 +54,7 @@ class _SupaEmailAuthState extends State<SupaEmailAuth> {
               return null;
             },
             decoration: const InputDecoration(
-              icon: Icon(Icons.email),
-              border: OutlineInputBorder(),
```

Comment:
By removing the border property here, the border style will match what is set in the theme of MaterialApp!